WooCommerce/Classes/Model/ShippingLabelAddress+Woo.swift

```diff
@@ -0,0 +1,72 @@
+import Foundation
+import Contacts
+import Yosemite
+
+
+// Yosemite.ShippingLabelAddress Helper Methods
+//
+extension ShippingLabelAddress {
+
+    /// Returns the `name`, `company`, and `address`. Basically this var combines the
+    /// various components of a ShippingLabelAddress.
+    ///
+    var fullNameWithCompanyAndAddress: String {
+        var output: [String] = [fullNameWithCompany]
+
+        if let formattedPostalAddress = formattedPostalAddress, formattedPostalAddress.isEmpty == false {
+            output.append(formattedPostalAddress)
+        }
+
+        return output.joined(separator: "\n")
+    }
+}
+
+private extension ShippingLabelAddress {
+
+    /// Returns the Postal Address, formated and ready for display.
+    ///
+    var formattedPostalAddress: String? {
+        return postalAddress.formatted(as: .mailingAddress)
+    }
+
+    /// Returns the `name` and `company` (on a new line). If either the `name` or `company` is empty,
+    /// then a single line is returned containing the other value.
+    ///
+    var fullNameWithCompany: String {
+        var output: [String] = []
+
+        if name.isEmpty == false {
+            output.append(name)
+        }
+        if company.isEmpty == false {
```

Comment:
I've seen that we sometimes use `var == false` and sometimes `!var` through our codebase.
Should we be following any style here, I search on our style guide and couldn't find any clear direction, WDYT?

Reply:
We don't have a specific rule for this, but since all these properties here are not optionals, I think that using `isNotEmpty` is the best option. I apply the changes here edd7cb2ce5b8695132689569879da78cf84bb7f8